src/main/groovy/com/calgaryscientific/gradle/XMLIO.groovy

```diff
@@ -98,6 +98,29 @@ class XMLIO {
         return null
     }
 
+    /**
+     * getNodeList - For a list of nodes names it will iterate over the first node
+     * matching the name until the last member in which case it will return the entire nodeList
+     */
+    static NodeList getNodeList(Node node, String... name) {
+        return getNodeList(node, name.toList())
+    }
+
+    static NodeList getNodeList(Node node, List<String> name) {
```

Comment:
what does this function do? do we have any examples on what this function returns?

Reply:
There is no unit test for this function. The doc is basically the same as:
https://github.com/calgaryscientific/veracode-gradle-plugin/pull/17/files#diff-c0f436b62d2752c07b86f08705f7636aR105

Reply:
can we have some examples on what this function would produce/generate? I'm not quite sure I understand what this function does....

Reply:
I added tests for XMLIO, please take a look at those.